bin/i18n/utils/malformed_i18n_reporter.rb

```diff
@@ -39,12 +38,11 @@ def process_file(i18n_file_path)
 
       def report
         return if worksheet_data.empty?
-
-        @google_drive&.update_worksheet(SPREADSHEET_NAME, locale, [WORKSHEET_HEADERS, *worksheet_data])
-
+        Google::Drive.new.update_worksheet(SPREADSHEET_NAME, locale, [WORKSHEET_HEADERS, *worksheet_data])
```

Comment:
Just want to confirm that reinitializing a google drive session every time this is invoked rather than saving and reusing the session is an intentional/acceptable change. I don't know of any reason in particular why it wouldn't be, just want to make sure we've considered that.